src/iso19111/c_api.cpp

```diff
@@ -192,6 +192,7 @@ getDBcontextNoException(PJ_CONTEXT *ctx, const char *function) {
 // ---------------------------------------------------------------------------
 
 PJ *pj_obj_create(PJ_CONTEXT *ctx, const BaseObjectNNPtr &objIn) {
+    SANITIZE_CTX(ctx);
```

Comment:
The definition of `SANITIZE_CTX` needs to be hoisted above this function. I'll do that later today.